commit 1f51ee9246b048d8966c36ddd2c26d7e0f927d83
Author: Steve McIntyre <steve.mcintyre@linaro.org>
Date:   Mon Nov 19 01:22:33 2012 -0500

    Add ldconfig cache tag handling for AArch64
    
    	* sysdeps/generic/ldconfig.h (FLAG_AARCH64_LIB64): New macro.
    	* elf/cache.c (print_entry): Print ",AArch64" for
    	FLAG_AARCH64_LIB64.
    
    Signed-off-by: Steve McIntyre <steve.mcintyre@linaro.org>
    Reviewed-by: Carlos O'Donell <carlos@systemhalted.org>

diff --git a/elf/cache.c b/elf/cache.c
index 3336bab..f5ed370 100644
--- a/elf/cache.c
+++ b/elf/cache.c
@@ -97,6 +97,9 @@ print_entry (const char *lib, int flag, unsigned int osversion,
     case FLAG_ARM_LIBHF:
       fputs (",hard-float", stdout);
       break;
+    case FLAG_AARCH64_LIB64:
+      fputs (",AArch64", stdout);
+      break;
     case 0:
       break;
     default:
diff --git a/sysdeps/generic/ldconfig.h b/sysdeps/generic/ldconfig.h
index a805284..43cb9bd 100644
--- a/sysdeps/generic/ldconfig.h
+++ b/sysdeps/generic/ldconfig.h
@@ -35,6 +35,7 @@
 #define FLAG_MIPS64_LIBN64	0x0700
 #define FLAG_X8664_LIBX32	0x0800
 #define FLAG_ARM_LIBHF		0x0900
+#define FLAG_AARCH64_LIB64	0x0a00
 
 /* Name of auxiliary cache.  */
 #define _PATH_LDCONFIG_AUX_CACHE "/var/cache/ldconfig/aux-cache"
